rules/windows/privilege_escalation_printspooler_suspicious_spl_file.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
```

Comment:
```suggestion
ecs_version = ["1.6.0"]
```

Reply:
I think this version does need to be `1.6.0`